### PR TITLE
Add: kafka setup for catalogue service

### PR DESCRIPTION
**Pull request by luco5826**
This PR sets up Kafka for the **TicketCatalogueService**

Waiting for #11 to be merged before adding db interaction and POST method (#3 )

TODO:
- DB interaction for the **PaymentResponse** consumer (update the DB with the result from **PaymentService**
- POST `/shop/<ticket-id>` controller (interaction with **TravelerService** and **PaymentService** through Kafka

### CODE REVIEW

**Comment by fxbrit** [comment]
as part of this PR let's also get rid of [paymentservice/controller/TestingController.kt](https://github.com/Ale142/group03-reactive-kafka/blob/master/PaymentService/src/main/kotlin/it/polito/wa2/paymentservice/controller/TestingController.kt) as we'll no longer need it.

**Review by riccardomengoli — APPROVED** [review]

File: PaymentService/src/main/kotlin/it/polito/wa2/paymentservice/kafka/KafkaConfig.kt [hunk 1]
```diff
@@ -36,7 +36,7 @@ class KafkaConfig(@Value("\${spring.kafka.bootstrap-servers}") private val serve
     /** outgoing payment responses from PaymentService to other internal services. */
     @Bean
     fun paymentToTraveler(): NewTopic {
```
[the remark this answers is not in the document]
Comment (fxbrit) [reply PaymentService/src/main/kotlin/it/polito/wa2/paymentservice/kafka/KafkaConfig.kt:38]: fixed in c80ed74b46a7297384777bacdf0f87e3364e4234

[the remark this answers is not in the document]
Comment (luco5826) [reply PaymentService/src/main/kotlin/it/polito/wa2/paymentservice/kafka/KafkaConfig.kt:38]: Done with c80ed74b46a7297384777bacdf0f87e3364e4234

[the remark this answers is not in the document]
Comment (fxbrit) [reply PaymentService/src/main/kotlin/it/polito/wa2/paymentservice/kafka/KafkaConfig.kt:38]: bruh

**Pull request merged**